Casks/k/keyguard.rb

```diff
@@ -0,0 +1,35 @@
+cask "keyguard" do
+  version "r20241002,1.6.0"
+  sha256 "3631112e36eb33d5b9c7d2019bd418650c777d98b1fe2af1f9797e47364bf0ef"
+
+  url "https://github.com/AChep/keyguard-app/releases/download/#{version.csv.first}/Keyguard-#{version.csv.second}.dmg"
```

Comment:
```suggestion
  version "1.6.0,20241002"
  sha256 "3631112e36eb33d5b9c7d2019bd418650c777d98b1fe2af1f9797e47364bf0ef"

  url "https://github.com/AChep/keyguard-app/releases/download/r#{version.csv.first}/Keyguard-#{version.csv.second}.dmg"
```

We should list the app version first.

Reply:
https://github.com/AChep/keyguard-app/releases/download/r20241002/Keyguard-1.6.0.dmg

But the URL pattern is like. Won't above changes break the URL ?

Reply:
Thanks, that was an oversight on my part. I have updated the suggestion.